incentivizing_rewarding_participants.adoc

```diff
@@ -1,87 +1,129 @@
+Community managers therefore shouldn't strive to create experiences _they believe_ will motivate everyone to join and contribute to the project as much as they should work to understand participants' _own_ motivations for contributing to projects, then architecting project experiences that activate and satisfy those motivations, leading (presumably) to greater engagement.
+This is critical not only because it puts community members and their needs first, but also because it allows contributors with varying degrees of time, resources, knowledge, and skill to engage a project in ways that feel comfortable for them—and feel rewarded in just the same ways others might.
+
+== Understanding a project's reward models
+
+Strategies for acknowledging and honoring various contributions to an open source project are called that project's "reward models."
+The more reward models your project constructs, the better it can engage participants with differing motivations for contributing.
+As we've already noted, everyone contributes to projects for different reasons, and the most popular projects are those that make contributors feel rewarded, valued, and respected for their contributions—whatever form they happen to take.
+Generating those feelings in contributors is a great way to build your project's reputation as a rewarding, welcoming place.
+But always remember that your community's reward models will _incentivize_ the very practices they _reward_; when people understand how a community recognizes and honors its participants, they'll engage in the behaviors they observe a community celebrate in this way.
+Community managers and architects should therefore always think carefully and act deliberately when establishing and modifying a project's particular reward model.
+What they reward—either intentionally or unintentionally—is what they'll be encouraging people to do more of.
+
+In what remains of this chapter, we'll outline some of the more popular reward models open source projects use today.
+As each of these models aims to connect with a certain set of motivations participants may have for contributing to a project, we'll elaborate situations in which a particular model might be most useful.
+
+=== Point, badge, and leaderboard systems
+
+Some open source communities feature point, badge, and leaderboard systems that allow participants to track and publicly display a history of contributions to a project.
+For example, a community might maintain a list of "top contributors" or contributors "most active this month" in order to generate lighthearted competition that spurs project contribution.
+To do this, it might assign point values to project activities it seeks to elicit (fix a typo in documentation, submit a patch, assist a new user in a forum, etc.), then reward people who engage in those activities with the corresponding points (which, when tallied, determine contributors' ranking on the community leaderboard).
```

Comment:
The point system is a good fit with leaderboards, but they need to be further distinguished i.e.

Point systems on their own
Leaderboards on their own
Similarities and differences between the two.

Reply:
Well put and easy to do. Thanks!